rtcstats.js

```diff
@@ -212,6 +212,16 @@ module.exports = function(wsURL, getStatsInterval, prefixesToWrap) {
         }
       });
 
+      ['addTrack', 'removeTrack'].forEach(function(method) {
```

Comment:
you can include those two method names in the array & iteration in line 191 (`['createDataChannel', 'close']`)